Platon/backend/app/search_engine/views.py

```diff
@@ -574,5 +577,99 @@ def get(self):
         else:
             return make_response(jsonify({"error": "Missing data fields or invalid data."}), 400)
 
+@search_engine_ns.route("/tag_search")
+class TagSearchAPI(Resource):
+
+    @api.doc(responses={401: 'Account Problems', 400: 'Input Format Error', 500: 'Database Connection Error'})
+    @api.response(200, 'Valid Users', search_user_list_model)
+    @api.response(201, 'Valid Workspaces', workspace_list_model)
+    @api.expect(tag_search_parser)
+    def get(self):
+        form = TagSearchForm(request.args)
+        if form.validate():
+            skill_list = json.loads(form.skills.data)
+            # Find the ids of the skills
+            try:
+                skill_ids = [Skills.query.filter_by(name=skill.title()).first().id for skill in skill_list]
+            except:
+                return make_response(jsonify({"error": "Database Connection Problem."}), 500)
```

Comment:
As far as I remember, tag search was expected to be performed after clicking to a specific tag in a workspace page or user profile page. I didn't understand why list of skills is used instead of only one skill.

Reply:
Actually, I implemented it in a more functional way. Because I do not have so much time next week, and I think that they can want to use this part with more than one skill(in an intersection way). So I implemented it like that. The functionality that you specify can be achieved by using a list whose length is 1.